tests/address-utils/test_address_utils.py

```diff
@@ -96,14 +122,18 @@ def test_is_checksum_formatted_address(value, expected):
 @pytest.mark.parametrize(
     "value,expected",
     [
-        # weird values
-        (lambda: None, False),
-        ("function", False),
-        ({}, False),
```

Comment:
Why delete these? It seems an unlikely scenario, but they're not slow tests or anything.